app/repository/RaceGroupRepositoryImpl.java

```diff
@@ -1,20 +1,21 @@
 package repository;
 
-import models.Judgment;
+import com.fasterxml.jackson.databind.JsonNode;
 import models.RaceGroup;
 import play.db.jpa.JPAApi;
 import repository.interfaces.RaceGroupRepository;
 
 import javax.inject.Inject;
 import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.persistence.NoResultException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)